app/models/equipment_profile.rb

```diff
@@ -12,15 +12,15 @@ class EquipmentProfile < ApplicationRecord
   before_validation :null_subcategory
   
   def full_category
-    if subcategory
-      "#{category} - #{subcategory}"
+    if self.subcategory
+      "#{self.category} - #{self.subcategory}"
     else
-      category
+      self.category
```

Comment:
This method is used in at least one view (`show` view for a profile). It seems to work fine on production Tracker in the view. Does not using `self` only affect validations?